x/examples/outline-connectivity-app/shared_frontend/pages/connectivity_test/index.ts

```diff
@@ -742,22 +752,41 @@ export class ConnectivityTestPage extends LitElement {
     </main>`;
   }
 
+  renderPlatformStyles() {
+    if (!this.platform) {
+      return nothing;
+    }
+
+    // TODO: changes how switches look on android
+    switch (this.platform.operatingSystem) {
```

Comment:
This looks a little bit inflexible. Is it possible to add a new class to the root element like `platform-ios`, and in the CSS we can have dedicated rule for this class?